docs/C-Data-Structures/structures.md

```diff
@@ -385,82 +283,25 @@ The output produced from the above sample is shown below:
 
 The table includes:
 
 * the structure type of each object
-* the identifer of each object
+* the identifier of each object
 * the type of each member
 * the identifier of each member
 
->Note the breakdown of each object into its members in the head of the table.  We reserve a separate line for the addresses of the different objects:
-
-<table border="0">
-<tr>
-    <td colspan="2">&nbsp;</td>
-    <td align="center" colspan="2">struct<br/>Product</td>
-    <td align="center" colspan="2">struct<br/>Product</td>
-    <td align="center" colspan="2">struct<br/>Product</td>
-    <td align="center" colspan="2">struct<br/>Product</td>
-</tr>
-<tr>
-    <td colspan="2">&nbsp;</td>
-    <td align="center" colspan="2">product[0]</td>
-    <td align="center" colspan="2">product[1]</td>
-    <td align="center" colspan="2">product[2]</td>
-    <td align="center" colspan="2">product[3]</td>
-</tr>
-<tr>
-    <td colspan="2">&nbsp;</td>
-    <td colspan="2">1000</td>
-    <td colspan="2">100C</td>
-    <td colspan="2">1018</td>
-    <td colspan="2">1024</td>
-</tr>
-<tr>
-    <td align="center">int</td>
-    <td align="center">int</td>
-    <td align="center">int</td>
-    <td align="center">double</td>
-    <td align="center">int</td>
-    <td align="center">double</td>
-    <td align="center">int</td>
-    <td align="center">double</td>
-    <td align="center">int</td>
-    <td align="center">double</td>
-</tr>
-<tr>
-    <td align="center">i</td>
-    <td align="center">n</td>
-    <td align="center">sku</td>
-    <td align="center">price</td>
-    <td align="center">sku</td>
-    <td align="center">price</td>
-    <td align="center">sku</td>
-    <td align="center">price</td>
-    <td align="center">sku</td>
-    <td align="center">price</td>
-</tr>
-<tr><td>&nbsp;</td>    <td align="center">4</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-</tr>
-<tr><td>&nbsp;</td>    <td align="center">4</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-</tr>
-<tr><td>&nbsp;</td>    <td align="center">4</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-</tr>
-<tr><td>&nbsp;</td>    <td align="center">4</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-</tr>
-<tr><td>&nbsp;</td>    <td align="center">4</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>    <td>&nbsp;</td>
-</tr>
-</table>
+:::note
+
+Note the breakdown of each object into its members in the head of the table.  We reserve a separate line for the addresses of the different objects:
```

Comment:
For the admonitions, don't repeat "Note" again... in this case state "The breakdown..."